src/game/client/tf/c_tf_player.cpp

```diff
@@ -3461,7 +3468,14 @@ void C_TFPlayer::CreatePlayerGibs( const Vector &vecOrigin, const Vector &vecVel
 
 	// Break up the player.
 	m_hSpawnedGibs.Purge();
-	m_hFirstGib = CreateGibsFromList( m_aGibs, GetModelIndex(), NULL, breakParams, this, -1 , false, true, &m_hSpawnedGibs );
+	if ( bBurning )
```

Comment:
Not sure why you pass the parameter through instead of just checking m_bBurning here?